src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,100 @@
+package wolox.training.models;
+
+import com.sun.istack.NotNull;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+/**
+ * Entity  for table User
+ */
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "USER_SEQ")
+    @SequenceGenerator(name = "USER_SEQ", sequenceName ="USER_SEQ")
+    private long id;
+    @NotNull
+    @Column(nullable = false)
+    private String username;
+    @NotNull
+    @Column(nullable = false)
+    private String name;
+    @NotNull
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    /**
+     * Field for relation user-book
+     */
+    @OneToMany(mappedBy = "user_id")
+    private List<Book> books;
+
+    public User() {
+        this.books = new ArrayList<>();
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
```

Comment:
The set is not necessary because it is self-increasing

Reply:
I deleted the set method